### PR TITLE
Update cookie policy

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

This PR updates and enables Cookie Policy page
https://docs.google.com/document/d/1VHltjyMR2ubWToWqdCzgyIsjl4nEhQEucwQ27HdYxBY/edit#

# To test
- go to `/cookie-policy` page
- make sure the content is correct as defined in the doc

### CODE REVIEW

File: src/cow-react/pages/CookiePolicy/CookiePolicy.md [hunk 1]
```diff
@@ -1,83 +1,128 @@
...
+
+   If you do not accept changes which are made to this Cookie Policy, or take any measures described above to opt-out by removing or rejecting cookies, you may continue to use this website but accept that it may not display and/or function as intended by us. Your exercise of any rights to opt-out may also impact how our information and content is displayed and/or accessible to you on this website and on other websites.
 
 ### APPENDIX
 
 _Table: Overview of cookies placed and the consequences if the cookies are not placed_
 
+![cookie-policy-1.png](https://raw.githubusercontent.com/cowprotocol/cowswap/088e9ef3f018804be9850c0fc65fa3eea26c88d6/src/custom/assets/cow-swap/cookie-policy/cookie-policy-1.png)
```
Comment (nenadV91) [inline src/cow-react/pages/CookiePolicy/CookiePolicy.md:122]: These links should be updated with image links from `main` branch once these images are there.

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @nenadV91 , great!

Some issues I faced: 
1. The same as for Privacy policy, it would be great to have a different icon for the Cookie policy in the menu
image

2. We should use the same format for all 'Last updated' sentences across all content pages (TC, PP, CP). So it would be great to make it look the same like on the Terms page, + add Last, remove parentheses: 
cookie last
last upd
3. I don't think we need an additional background area for paragraphs. I think, we need to use the same format for all content pages (no additional bg): 
no need in bg
4. Do we really need to highlight it in red? It is not highlighted in the doc:
not in red
5. There are no screenshots in Chrome + Win11, desktop version (but I'm able to see them on mobile view)
no tables
6. For me, it is difficult to scroll the table: long press selects and highlights the text, but does not scroll the table content. Can we add a scroll effect on the long press action? 
image
7. It would be great to enhance the scroll bar itself in Win11
image
8. The table in a mobile view does not look good: 
image
9. Some links exceed the screen width in a mobile view:
image

Thanks!

**Comment by alfetopito** [comment]
From Chris on [this thread](https://cowservices.slack.com/archives/C036239R2QL/p1672672303108169?thread_ts=1669300269.450959&cid=C036239R2QL)

> I don’t think the Cookie Policy needs the screenshot - this was only in the google doc to inform what the table needs to have included.

**Review by alfetopito — APPROVED** [review]
I've removed screenshots as requested by Chris

@nenadV91 please address remaining points raised by Elena when you are back


Will merge as is for now

**Comment by elena-zh** [comment]
> I've removed screenshots as requested by Chris
> 
> @nenadV91 please address remaining points raised by Elena when you are back
> 
> Will merge as is for now

I have created #1825 task for the issues I listed above.

**Pull request merged**